iscript/src/iscript/mac.py

```diff
@@ -926,7 +926,7 @@ async def create_pkg_files(config, sign_config, all_paths):
     for app in all_paths:
         # call set_app_path_and_name because we may not have called sign_app() earlier
         set_app_path_and_name(app)
-        app.pkg_path = app.app_path.replace(".appex", ".pkg").replace(".app", ".pkg")
+        app.pkg_path = app.app_path.replace(".appex", ".pkg").replace(".app", ".pkg").replace(".systemextension", ".pkg")
```

Comment:
Interesting, we want to create .pkg files for the system extension(s)?
If not, we could also skip `.systemextension` apps during pkg creation.

Reply:
We don't really want to (the very first thing we do with these is unpacking them to get the actual bits out), but this is what iscript spits out right now. Ideally (at least, IMO) we'd spit out the same format as the input.

Reply:
iscript spits out a tarball and a pkg. if we only extract it from the tarball, we can probably skip the pkg ?

Reply:
Oh god, I can't believe I missed that (pkgs are listed last in the log output, which is the only thing I saw). Yes, let's skip pkg generation for mozillavpn! I'll need to update the braindump scripts + this patch to do that.

Reply:
Ok. Non-blocker. We waste a tiny bit of compute time and a tiny bit of disk space/network bandwidth for the pkg, but if we end up landing as-is I'm ok with it.

Reply:
Too late, I already did this! I'll have a new version up for review shortly.